### PR TITLE
New activate contenttypes command

### PR DESCRIPTION
**Pull request by theus77**

### CODE REVIEW

File: Command/ActivateContentTypeCommand.php [hunk 1]
```diff
@@ -0,0 +1,147 @@
+<?php
+
```
Comment (Davidmattei) [inline Command/ActivateContentTypeCommand.php]:
```suggestion
declare(strict_types=1);
```
For new files we should start doing this

File: Command/ActivateContentTypeCommand.php [hunk 2]
```diff
@@ -0,0 +1,147 @@
...
+        if (!$input->getOption(self::OPTION_ALL) && count($types) == 0) {
+            $this->chooseTypes($input, $output);
+        }
+
+        if ($input->getOption(self::OPTION_ALL)) {
+            $this->optionAll($input);
+        }
+        $this->deactivate = $input->getOption(self::DEACTIVATE);
+        $this->force = $input->getOption(self::FORCE);
```
Comment (Davidmattei) [inline Command/ActivateContentTypeCommand.php]:
```suggestion
```
We are not interacting with these options, so you can just all getOption in the execute, also remove the 2 private properties 'deactivate' and 'force' on this command class

File: Command/ActivateContentTypeCommand.php [hunk 3]
```diff
@@ -0,0 +1,147 @@
...
+            )
+            ->addOption(
+                self::FORCE,
+                null,
+                InputOption::VALUE_NONE,
+                sprintf('Activate the contenttypes even if the mapping is not up to date (flagged as draft)')
+            );
+    }
+
+    public function execute(InputInterface $input, OutputInterface $output)
```
Comment (Davidmattei) [inline Command/ActivateContentTypeCommand.php]:
```suggestion
    protected function execute(InputInterface $input, OutputInterface $output)
```

Comment (theus77) [reply Command/ActivateContentTypeCommand.php]: execute as protected function

File: Command/ActivateContentTypeCommand.php (hunk 2 above)
Comment (theus77) [reply Command/ActivateContentTypeCommand.php]: Agree for the force but the deactivate option : see 30930983116d0cff3ff17bcd77bdedb57115827d

Comment (Davidmattei) [reply Command/ActivateContentTypeCommand.php]: Oh okay I see, you need the deactivate in the chooseTypes and interact function. Okay I can live with that :)

**Pull request merged**